cephadm-ansible.spec.in

```diff
@@ -0,0 +1,42 @@
+%global commit @COMMIT@
+%global shortcommit %(c=%{commit}; echo ${c:0:7})
+
+Name:           cephadm-ansible
+Version:        @VERSION@
+Release:        @RELEASE@%{?dist}
+Summary:        Ansible playbooks for Ceph
+License:        ASL 2.0 and GPLv3+
```

Comment:
What is the license for this project?

Reply:
The main ceph-ansible project is licensed as `ASL 2.0`, but there is a comment in `ceph-ansible.spec.in`:

```
# Some files have been copied from Ansible (GPLv3+). For example:
#  library/ceph_facts
#  plugins/actions/config_template.py
#  roles/ceph-common/plugins/actions/config_template.py
```

So if we're copying files from Ansible into `cephadm-ansible`, we need to indicate `GPLv3+`, but if we're only copying files from `ceph-ansible` or writing new files, we can go with `ASL 2.0`.

Reply:
@ktdreyer I don't think we are going to copy files from Ansible, can we leave ASL 2.0 *and* GPLv3+ anyway?

Reply:
Let's just keep it ASL 2.0 then for simplicity.

https://docs.fedoraproject.org/en-US/packaging-guidelines/LicensingGuidelines/#_dual_licensing_scenarios explains more about Dual Licensing and Multiple Licensing.